### ISSUE TITLE
Traits-like Abilities Documentation

### ISSUE THREAD

**Issue by CelticMinstrel** [CelticMinstrel/wesnoth#228]
**[Original submission](https://gna.org/bugs/index.php?17432)** by gallaecio on 2011-01-07

I would like to ask for Abilities to be managed as traits in Wesnoth help. Currently, abilities male form is used as title, and ability description's first line is ignored (see screenshots).

Instead, as it's done with traits, it would be good to have male/female name (for ingame, as adjective), general name (to be used as title in the Help) and description, without ability "general name" inside it.

In short: abilities documentation managed the same way as trait's one (from a l10n point of view).

**Comment by CelticMinstrel** [comment]
**Modified on 2011-01-07**

**gallaecio added attachment:** [wesnoth.png](http://gna.org/bugs/download.php?file_id=11726)

**gallaecio added attachment:** [wesnoth2.png](http://gna.org/bugs/download.php?file_id=11727)
